apps/web/src/app/(signing)/sign/[token]/page.tsx

```diff
@@ -56,7 +56,11 @@ export default async function SigningPage({ params: { token } }: SigningPageProp
   }
 
   return (
-    <SigningProvider email={recipient.email} fullName={recipient.name} signature={user?.signature}>
+    <SigningProvider
+      email={recipient.email}
+      fullName={user?.email === recipient.email ? recipient.name : undefined}
```

Comment:
So for this one we probably want something like `user?.email === recipient.email ? user.name : recipient.name` since we should use the name that the author has specified while allowing the recipient to change it if necessary.

Reply:
ok.. no problem